pages/how-to-contribute.md

```diff
@@ -62,10 +63,13 @@ Any questions/ideas don't hesitate to [contact us][contact].
 
 * **Build resources** - [Jenkins][jenkins]
 * **Releases** - [Making a release][making], [Verifying a release][verifying]
+* **Confluence** - [Accumulo Confluence Space][confluence] provides a collaboration space for design documents.  If you would like write
+access, please send a request to `dev@accumulo.apache.org` with your Confluence username.
```

Comment:
I would characterize this as a "require" rather than a "want", because there should be an actual need. Characterizing it as a need, implies that the requester provide some answer to "why?" up front, rather than us asking it as a follow-up question. So, instead of "I want write access" (and us having to ask "for what?"), it becomes "I require write access because I want to contribute to document X".

```suggestion
* **Confluence** - [Accumulo Confluence Space][confluence] provides a collaboration space for design documents. If you require write
access, please [contact us][contact] with your Confluence username and request details.
```